scripts/update/completion.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+#
+# Set the correct permissions for the completion file and then create a symlink to the bash completion file.
+chmod 644 '/etc/swizzin/sources/bash_completion.d/swizzin'
+ln -fs '/etc/swizzin/sources/bash_completion.d/swizzin' '/etc/bash_completion.d/swizzin'
```

Comment:
```suggestion
if [ ! -e /etc/bash_completion.d/swizzin ]; then
    chmod 644 '/etc/swizzin/sources/bash_completion.d/swizzin'
    ln -s '/etc/swizzin/sources/bash_completion.d/swizzin' '/etc/bash_completion.d/swizzin'
    echo_info "Shell completions for the box command have been installed, try pressing <tab> the next time you type 'box'" 
fi
```

Reply:
I don't think there is a need to check it exists, it's a symlink. just recreate it and we are done